scripts/bindings.py

```diff
@@ -37,15 +38,16 @@ def generic_head():
     :return: string
     """
 
-    return (
-        '"""\n'
-        ' Python wrapper for libui.\n'
-        '\n'
-        '"""\n'
-        '\n'
-        'import ctypes\n'
-        'from . import clibui\n\n\n'
-    )
+    return '''"""
+ Python wrapper for libui.
+
+"""
+
+import ctypes
+from . import clibui
+
+
+'''
```

Comment:
I've mixed feelings regarding lack of indentation on the code base. In this case it is a "visual" thing, although Python is incoherent in these cases..